onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -6014,10 +6014,93 @@ def aten_var(self: TensorType, unbiased: bool = True) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_var_mean(self: TensorType, unbiased: bool = True) -> tuple[TensorType, TensorType]:
+@torch_op("aten::var_mean", trace_only=True)
+def aten_var_mean(self: TReal, unbiased: bool = True) -> tuple[TReal, TReal]:
     """var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)"""
 
-    raise NotImplementedError()
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean(self, correction, keepdim=False)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_dim(
+    self: TReal, dim: int, unbiased: bool = True, keepdim: bool = False
```

Comment:
```suggestion
    self: TReal, dim: Optional[int], unbiased: bool = True, keepdim: bool = False
```

according to the signature below?

Reply:
https://pytorch.org/docs/master/generated/torch.var_mean.html?highlight=var_mean#torch.var_mean
PyTorch document say correction is int.
I tried to change the signature to float, the FullGraph mode has some trouble for:
```
if correction != 0.0
```
Because the input correction is 2 instead of 2.0